instructure/canvas-android
# [MBL-16957][Student] Offline LGO support

Test plan: 
- Test all the screens where LGO has an effect.
- Verify that it works both in offline and online mode.
- Verify that scores are correctly displayed when LGO is turned off.

refs: MBL-16957
affects: Student
release note: none

## Checklist
